### PR TITLE
Add a doc for how to use fuse node selector

### LINKED ISSUE #1470: [DOC] Missing doc for how to use fuse node selector
**Provide a link to that doc page:**

**What is the defect and your suggestions on improvement:**
Missing doc for how to use fuse node selector.

We should present a document to guide users how to use fuse node selector. A document with a sample or a demo is recommended.

Here's my suggestion for the doc:
1. Setup an environment with two Kubernetes Nodes.
2. Add a label to one of the nodes
3. Create a Dataset and an AlluxioRuntime with `spec.fuse.nodeSelector` set to the label just added.
4. Create two pods with podAntiAffinity for each other. 
5. Expected result: one pod should be scheduled to the Node with the added label, and the other pod should be pending since `spec.fuse.nodeSelector` prevent it from being scheduled to another Node.

### PR DESCRIPTION
**Pull request by haoeeeee**
Signed-off-by: wangxu <wangxu199811@163.com>

 Ⅰ. Describe what this PR does
Add a doc for how to use fuse node selector

Ⅱ. Does this pull request fix one issue?
fixes #1470 :[DOC] Missing doc for how to use fuse node selector

### CODE REVIEW

File: docs/en/samples/fuse_affinity.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+# 示例 - Fuse NodeSelector使用
+
+在Fluid中，`Dataset`资源对象中所定义的远程文件是可被调度的，这意味着你能够像管理你的Pod一样管理远程文件缓存在Kubernetes集群上的存放位置。 而执行计算的Pod可以通过Fuse客户端访问数据文件。Fuse客户端可以在Kubernetes集群中全局部署,我们可以通过`nodeSelector`来指定Fuse客户端的部署范围。
+
```
Comment (cheyang) [inline docs/en/samples/fuse_affinity.md:4]: `指定Fuse客户端的部署范围` -> `限制Fuse客户端的部署范围`

Comment (cheyang) [inline docs/en/samples/fuse_affinity.md:4]: `指定Fuse客户端的部署范围` -> `限制Fuse客户端的部署范围`

Comment (haoeeeee) [reply docs/en/samples/fuse_affinity.md:4]: Ok, I will revise this part, and I just found out that the file I submitted was in the wrong directory. Should I resubmit both Chinese and English documents？

Comment (cheyang) [reply docs/en/samples/fuse_affinity.md:4]: Feel free to move the file to the correct directory. And it's fine to commit the doc in Chinese only. You are welcome to create another PR for English version. Thanks.

**Pull request closed without merging**